src/main/java/com/sapozhnykov/services/order/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.sapozhnykov.services.order.impl;
+
+import com.sapozhnykov.dao.order.OrderDao;
+import com.sapozhnykov.dao.order.impl.OrderDaoImpl;
+import com.sapozhnykov.domain.Order;
+import com.sapozhnykov.domain.Product;
+import com.sapozhnykov.services.order.OrderService;
+
+import java.util.ArrayList;
+
+public class OrderServiceImpl implements OrderService {
+    private OrderDao orderDao = new OrderDaoImpl();
+
+    @Override
+    public boolean add(ArrayList<Product> products) {
+        /* We don't have a Session to get client ID.
```

Comment:
You have Client

Reply:
I added authorization to determine user ID